inject.js

```diff
@@ -125,7 +125,12 @@
       if (target.dataset['vscid']) {
         return target.vsc;
       }
-
+      this.zindex = target.style.zIndex;
+      if (this.zindex === "") {
```

Comment:
zindex can be a string other than empty string: https://www.w3schools.com/jsref/prop_style_zindex.asp

- Would setting to 1 potentially cause issues when initial value is not empty string?
- If we set the value to "inherit" which should be the z-index of the video element, would that work?

Reply:
Thanks for the thoughts... I'm going to try using inherit and see how that performs. I didn't realize that was an option, thanks for linking the docs!

Reply:
I really like how much cleaner it is, but it seems using "inherit" is only working for Vimeo for some reason. It is not working for YouTube, LinkedIn Learning, or Viki in my quick tests.

I haven't put a lot of effort into investigating why at this point. I need to get going for now but I will revisit and reply once I've had the chance to look at it more closely.

Reply:
As best as I can tell, the issue with inherit not working as intended seems to stem from it using the same z-index as the element.

In turn, the final choice as to whether it's shown or not depends on what I infer is [the rules governing the way "auto" works](https://www.w3schools.com/cssref/pr_pos_z-index.asp) - "Sets the stack order equal to its parents".

I've come up with a [pretty comprehensive solution](https://github.com/igrigorik/videospeed/pull/539/commits/4ca7611529def6ebacf094c33d96892a783a55c1) that seems to be accounting for every scenario and ultimately making the z-index 1-level higher than the video. This does produce some interesting results in edge cases. For example, here is a facebook story video screenshot:

![image](https://user-images.githubusercontent.com/15102816/68065836-da8ad300-fcfc-11e9-9a39-bad1dcb35597.png)

This twitch screenshot shows the same scenario, but with expected(?) results:
![image](https://user-images.githubusercontent.com/15102816/68066096-926daf80-fd00-11e9-9183-7c98aa464b84.png)


I don't think this is avoidable as this turns in to a matter of opinion. That is to say, how would one know when to render over or under the blocking element? In my opinion, making it 1 layer higher than the video element is an acceptable outcome, but I certainly can only speak for myself.